free-programming-books-ru.md

```diff
@@ -300,7 +300,7 @@
 
 * [Онлайн-руководство по AngularJS](https://metanit.com/web/angular) - Евгений Попов
 * [Перевод документации](http://angular-doc.herokuapp.com)
-
+* [Русская версия документация Angular 10](https://angular24.ru/start) - Алексей Охрименко
 
```

Comment:
There should be 2 line breaks between each section in the document.

Reply:
Yeah, you right. Sorry, I've just fixed it.

Reply:
Something is weird. I received the issue from Travis CI the build is failed, but I didn't change the file free-programming-books/free-programming-books-ta.md

Reply:
Don't mind that, I haven't checked fully, but it looks like a commit that screwed the formatting was merged with master. 

This PR should fix that issue for everyone:  
https://github.com/EbookFoundation/free-programming-books/pull/4253

Reply:
Okay. Thank you. Only one question how is it possible? I mean how someone managed to merge a broken commit to master.

Reply:
Maintainers are allowed to merge PRs, even if a check is failing.

It's handy in case a check is failing for known reasons unrelated to the PR. 

Even this PR can be merged by a maintainer already, even though checks are failing.

Reply:
Okay, I got it.